src/Task/Deptrac.php

```diff
@@ -59,16 +47,10 @@ public function run(ContextInterface $context): TaskResultInterface
         }
 
         $arguments = $this->processBuilder->createArgumentsForCommand('deptrac');
-        $arguments->add('analyze');
-        $arguments->addArgumentArray('--formatter=%s', $config['formatter']);
-        $arguments->add('--graphviz-display='.(int) $config['graphviz_display']);
-        $arguments->addOptionalArgument('--graphviz-dump-image=%s', $config['graphviz_dump_image']);
-        $arguments->addOptionalArgument('--graphviz-dump-dot=%s', $config['graphviz_dump_dot']);
-        $arguments->addOptionalArgument('--graphviz-dump-html=%s', $config['graphviz_dump_html']);
-        $arguments->addOptionalArgument('--junit-dump-xml=%s', $config['junit_dump_xml']);
-        $arguments->addOptionalArgument('--xml-dump=%s', $config['xml_dump']);
-        $arguments->addOptionalArgument('--baseline-dump=%s', $config['baseline_dump']);
-        $arguments->addOptionalArgument('%s', $config['depfile']);
+        $arguments->add('analyse');
+        $arguments->addOptionalArgument('--formatter=%s', $config['formatter']);
```

Comment:
This would be comma separated version of the provided list of formatters

Reply:
Since there can be only one formatter, here all right.